ros/models.py

```diff
@@ -25,3 +25,30 @@ def display_performance_score(self):
             # 'io_score': self.performance_score['io_score'] // 20
         }
         return display_performance_score
+
+
+class System(db.Model):
+    __tablename__ = 'systems'
+    id = db.Column(db.Integer, primary_key=True)
+    account_id = db.Column(db.Integer)
+    inventory_id = db.Column(UUID(as_uuid=True), nullable=False)
+    __table_args__ = (db.UniqueConstraint('inventory_id'),)
+
+
+class Report(db.Model):
+    __tablename__ = 'reports'
+    system_id = db.Column(db.Integer, primary_key=True)
+    reported = db.Column(db.Date, default=date.today(), primary_key=True, nullable=False)
+    __table_args__ = (
+        db.ForeignKeyConstraint(['system_id'], ['systems.id']),
+        db.PrimaryKeyConstraint('system_id', 'reported'),
+    )
+
+
+class RhAccount(db.Model):
+    __tablename__ = 'rh_accounts'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.Text, nullable=False)
```

Comment:
@semtexzv should we have `account_number` column instead of `name` here?

Reply:
Probably a better name, yes.

Reply:
done.